src/test/resources/pages/studentHomeHTML.html

```diff
@@ -31,6 +31,29 @@ <h1>
   </div>
   <br>
   <br>
+  <div class="panel panel-primary">
```

Comment:
If you revert all the changes in this HTML file you should get a passing Travis build.

Reply:
Ok! I'll give that a shot. I'm sorry that this pull chain looks so awful at the moment.